### PR TITLE
Updated to version 3.3.3

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

Updated the version to 3.3.3 for a hot fix release

### CODE REVIEW

**Comment by kirankumarkolli** [comment]
Next release is 3.4.0. Master is supposed to have next version no?

**Comment by j82w** [comment]
We have been keeping master on the current version. We haven't been bumping the version until we do the release.

File: changelog.md [hunk 1]
```diff
@@ -141,6 +145,7 @@ Any requests to Azure Cosmos DB using a retired SDK are rejected by the service.
 
 | Version | Release Date | Retirement Date |
 | --- | --- | --- |
+| [3.3.2](#3.3.3) |October 29, 2019 |--- |
```
Comment (kirankumarkolli) [inline changelog.md]: Change to 3.3.3

**Comment by kirankumarkolli** [comment]
Do we need to wait till 3.3.3 is published?

**Pull request merged**